src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -475,7 +477,7 @@ def ensure_namespace_cleanup():
                                          raise_error=False)
 
 
-def delete_arc_agents(release_namespace, kube_config, kube_context, helm_client_location, is_arm64_cluster, no_hooks=False):
+def delete_arc_agents(release_namespace, kube_config, kube_context, helm_client_location, is_arm64_cluster=False, no_hooks=False):
```

Comment:
LGTM. Please test this change with connect and delete command for sanity if not done already

Reply:
Tested, working fine.